ReleaseProcess.md

```diff
@@ -0,0 +1,40 @@
+# WebLogic Deploy Tooling Project Release Process
+This document describes the process that should be followed to create a WebLogic Deploy Tooling (WDT) release.
+
+## Prerequisites
+- A local installation of WebLogic Server 12.2.1.x must be available.
+- The person running the release process must have admin privileges on the [WebLogic Deploy Tooling GitHub repo](https://github.com/oracle/weblogic-deploy-tooling) since the release process pushes to the master branch directly.
+- The person running the release process needs to create a GitHub Personal Access Token for the repository with (at least) the `repo:status`, `repo_deployment`, `public_repo`, and `security_events` privileges.
+- The person running the release needs a server added to their Maven `settings.xml` where the GitHub Personal Access Token is stored, as shown below.  Note that this token can either be stored in plain text of encrypted using [Maven password encryption](https://maven.apache.org/guides/mini/guide-encryption.html).
+
+```xml
+  <servers>
+    <server>
+      <id>github</id>
+      <passphrase>store plain text or encrypted token here</passphrase>
+    </server>
+  </servers>
+```
+
+## Software Release Process
+The best practice is to write the release notes that will be published to GitHub prior to starting the steps below.
+
+1. Set (and export) the environment variable `WLST_DIR` to `<WLS-install-dir>/oracle_common/common/bin`, replacing `<WLS-install-dir>` with the full path to the WLS 12.2.1.x installation directory.
+2. In the weblogic-deploy-tooling project directory, create a file called release.properties with content similar to the example shown below.  Note that the example is configured to cut the 1.9.11 release.
```

Comment:
add reverse tick emphasis to filename, `release.properties`

Reply:
done